src/templates/EditorialNewsletter/web/index.js

```diff
@@ -34,6 +34,7 @@ const createSchema = ({ ...args } = {}) => {
     List,
     ListItem,
     ListP,
+    Link: A,
```

Comment:
Using this causes the app to crash: `Error: React.Children.only expected to receive a single React element child.`
Any Idea as to why? @tpreusse